app/components/funeralpromo/FuneralPromoFooter.js

```
@@ -34,11 +34,13 @@ const FuneralPromoFooter = () => {
           <ul className='flex text-sm md:text-[14px] leading-[24px] font-normal gap-4 md:gap-[30.5px] text-[#DAEBF1] items-center'>
             <li className='hidden md:block'>Pišite nam</li>
             <li>
+              <a href='https://www.facebook.com/people/Osmrtnicacom/61581688912510/' target='_blank'>
               <img 
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)